test/common/cdp.py

```diff
@@ -42,14 +42,24 @@ def browser_path_chromium():
 
     Exit with an error if none is found.
     """
-    g = glob.glob("/usr/lib*/chromium-browser/headless_shell")
-    if g:
-        return g[0]
-
-    p = subprocess.check_output("which chromium-browser || which chromium || which google-chrome || true",
-                                shell=True, universal_newlines=True).strip()
-    if p:
-        return p
+    def headless_shell():
+        g = glob.glob("/usr/lib*/chromium-browser/headless_shell")
+        if g:
+            return g[0]
+
+    def standard_chrome():
+        p = subprocess.check_output("which chromium-browser || which chromium || which google-chrome || true",
+                                    shell=True, universal_newlines=True).strip()
+        if p:
+            return p
+
+    browser_types = [standard_chrome, headless_shell]
+    if show_browser is False:
```

Comment:
`if not show_browser`. Also, why would it pick `headless_shell` for `show_browser`? IMHO it's cleaner to fail with a "not found" error than silently running headless.